src/role_carry.js

```diff
@@ -37,8 +37,8 @@ roles.carry.preMove = function(creep, directions) {
       var source = creep.room.memory.position.creep[targetId];
       // TODO better the position from the room memory
       if (source !== null) {
-        let returnCode = creep.moveTo(Game.getObjectById(targetId));
-        if (creep.pos.getRangeTo(Game.getObjectById(targetId)) > 1) {
+        let returnCode = creep.moveTo(source);
+        if (creep.pos.getRangeTo(source) > 1) {
```

Comment:
not sure about that because it will not move until creep is in the room of targetId i believe. you must use targetRoom in the source define ( which isn't a source but a sourcer or a storage/extension for helpers) plus it's not for RCL <3 but for RCL >3 out of base.
Ofc, if you tried it and it work, i said nothing ^^'